company/values.md

```diff
@@ -1,23 +1,24 @@
 # Sourcegraph values
 
-These values are some of the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
+These values are the beliefs and principles that help us achieve our [goals](goals/index.md) and [vision](strategy.md#vision).
 
-This list isn't intended to cover everything we care about; instead, it lists the values that we frequently find useful and refer to. We'll keep this list up to date with the frequently used beliefs and principles (adding, editing, and removing entries as needed). Our hope is that this makes this list more accurate and useful than if it were a list of stale, vague, aspirational, or obvious values.
+- [**Be customer driven**](#be-customer-driven): You proactively work on the right things by continuously orienting your goals around delivering value to your customer.
+- [**Work as a team**](#work-as-a-team): You work collaboratively with your peers, cross-functional teammates, and leadership to create shared success, trust, and belonging.
+- [**High agency**](#high-agency): You have the power and the responsibility to improve Sourcegraph as a company and as a product. You deliver regardless of the circumstances.
+- [**High quality**](#high-quality): Every person on our team is individually responsible for knowing what high-quality work looks like and producing high-quality work.
```

Comment:
What happens when two teammates don't have the same perception of what high quality work means? Who's ultimately "right"?

I don't think we should expect every teammate to KNOW what high-quality work is in every situation — there's a lot of context dependent, and team dependent, judgment, that requires alignment and conversation. I feel like this is more of a "we" thing than an I thing.

I'd like people to have strong opinions, weakly held, willing to learn something new and commit to a team decision that follows honest discussion.

Reply:
After reading the correspondent section below, I'm suggesting this change.
```suggestion
- [**High quality**](#high-quality): Every person on our team is individually responsible for finding out what high-quality work looks like and producing that high-quality work.
```